pandas/tests/series/test_combine_concat.py

```diff
@@ -116,8 +116,35 @@ def test_update(self):
         df = DataFrame([{"a": 1}, {"a": 3, "b": 2}])
         df['c'] = np.nan
 
-        # this will fail as long as series is a sub-class of ndarray
-        # df['c'].update(Series(['foo'],index=[0])) #####
```

Comment:
Series ceased being a subclass of ndarray in 0.13, so this test has been uncommented for quite some time. Now it's finally time for it to shine. ;-)